### PR TITLE
Add Core

### PR DESCRIPTION
**Pull request by TelmaRF**
hi team, please review our code

### CODE REVIEW

File: libraries/authentication/bearer.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+class Bearer(Authorization):
+    """Authorization using bearer token method"""
+
+    def authentication(self, token) -> tuple:
+        """Returns session authorization using bearer token method"""
+        params = {}
+        session = requests.Session()
+        session.headers.update({'Authorization': 'Bearer ' + token})
+        return session, params
+
```
Comment (daniskuvc) [inline libraries/authentication/bearer.py:28]: Additional spaces should be removed

**Review by daniskuvc — CHANGES_REQUESTED** [review]
Additional spaces should be removed

**Review by Rogerrenjifo — CHANGES_REQUESTED** [review]
Please review no more than 80 characters per line.

**Review by Rogerrenjifo — APPROVED** [review]
Good Job Team 👍

File: libraries/authentication/bearer.py (hunk 1 above)
Comment (TelmaRF) [reply libraries/authentication/bearer.py]: Thanks! I changed your comments

**Review by daniskuvc — APPROVED** [review]
Thanks Telma, great job!

**Pull request merged**